pkg/csv2lp/points.go

```diff
@@ -0,0 +1,2895 @@
+// Original source from models/points.go in "github.com/influxdata/influxdb/"
```

Comment:
Is this code modified at all from what is in the influxes repo?

Reply:
I believe not at all, just contains points.go and 2 or 3 other smaller files that had utility functions that were used.